app/svelte/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": ".",
```

Comment:
Use should set `:rootDir": "./src",` otherwise `dist` folder generated during TS "compilation" is not as expected:
![image](https://user-images.githubusercontent.com/4112568/68549389-a64b9c80-03f7-11ea-9c8a-ad1eb7ae10af.png)

In order to  be able to do that you have to use `const packageJson = require('../../package.json');` in `src/server/options.ts` instead of `import packageJson from '../../package.json';`. As it is done for `react` for instance: https://github.com/storybookjs/storybook/blob/9fb72175b76b7ec613a34d31c15329a5333ab3b6/app/react/src/server/options.ts#L1

Reply:
BTW, not related at all to your PR, there is a weird thing with `*.d.ts` files, they are compiled to JS files while they shouldn't. I will take a look when I have time to. cc @ndelangen @kroeder

Reply:
Got it! Thanks for your advice and tips, I am learning a lot :)